src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,46 @@
+.watch {
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $watch-size;
+  height: $watch-size;
```

Comment:
`@include figureSize($watch-size, $watch-size);`
You may use function or mixin here